src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -930,50 +929,66 @@ def get_test_desc(self, test: str) -> str:
         except AttributeError:
             return test
 
-    def get_default_app_pipe_name(self) -> str:
-        return self.app_pipe
-
-    def write_to_app_pipe(self, command_dict: dict, app_pipe_name: Optional[str] = None):
+    def write_to_app_pipe(self, command_dict: dict, app_pipe_prefix: Optional[str] = None, app_pid: Optional[int] = None):
         """
-        Sends an out-of-band command to a Matter app.
+        Send an out-of-band command to a Matter app.
+        Args:
+            command_dict (dict): dictionary with the command and data
+            app_pipe_name (Optional[str], optional): Name of the cluster pipe file (i.e. /tmp/chip_all_clusters_fifo_ or /tmp/chip_rvc_fifo_). Defaults to None.
```

Comment:
Add details of what `None` means. I would remove the types from here ... I can read the type from the signature and things like `(Optional[str], optional)` seems doubly redundant.